frontend/src/metabase/parameters/utils/mapping-options.unit.spec.js

```diff
@@ -104,14 +104,59 @@ describe("parameters/utils/mapping-options", () => {
         );
         expect(options).toEqual([
```

Comment:
It now returns all `date` fields from the `order` table with correct section names.